client/my-sites/exporter/export-card/select.jsx

```diff
@@ -77,7 +77,7 @@ const mapStateToProps = ( state, ownProps ) => {
 	const { siteId, postType, fieldName } = ownProps;
 
 	const options = getPostTypeFieldOptions( state, siteId, postType, fieldName );
-	const value = getPostTypeFieldValue( state, siteId, postType, fieldName );
+	const value = getPostTypeFieldValue( state, siteId, postType, fieldName ) || '';
```

Comment:
Just a small suggestion - I'd put this fallback inside the React component itself (ie L67), as that's where the warning is generated.